zingocli/Cargo.toml

```diff
@@ -25,15 +24,17 @@ tower-http = { version = "0.2", features = ["add-extension"] }
 webpki-roots = "0.21.0"
 zingolib = { path = "../zingolib/" }
 zingoconfig = { path = "../zingoconfig/" }
+zingo-testutils = { path = "../zingo-testutils" }
+
+json = "0.12.4"
 
 [dev-dependencies]
+zcash_primitives = { workspace = true }
```

Comment:
this also moved ?

Reply:
Yes.. it's only used in test.   The `[dev-dependencies]` section (implemented as a toml table) of the manifest, is the right spot for test-only dependencies.